src/PanelTraits/Update.php

```diff
@@ -38,26 +40,25 @@ public function update($id, $data)
     /**
      * Get all fields needed for the EDIT ENTRY form.
      *
-     * @param  [integer] The id of the entry that is being edited.
-     * @param int $id
+     * @param int $id The id of the entry that is being edited.
      *
-     * @return [array] The fields with attributes, fake attributes and values.
+     * @return array The fields with attributes, fake attributes and values
      */
     public function getUpdateFields($id)
     {
         $fields = $this->update_fields;
         $entry = $this->getEntry($id);
 
-        foreach ($fields as $k => $field) {
+        foreach ($fields as &$field) {
```

Comment:
It's not really immutable if you end up changing the array the same way in the end :D.
Agree with the 'readable' part though. I'll see what I can do.